packages/trader/src/Assets/Trading/Categories/icon-trade-categories.jsx

```diff
@@ -223,6 +226,9 @@ const IconTradeCategory = ({ category, className }) => {
                         <div className='category-wrapper'>
                             <Icon icon='IcTradetypeTurboslong' className='category-type' color='brand' />
                         </div>
+                        <div className='category-wrapper'>
```

Comment:
Isn't cases for `turbosshort` and `turboslong` similar? Maybe you can unite them in one case

Reply:
`case 'turboslong' || 'turbosshort':
                IconCategory = (
                    <React.Fragment>
                        <div className='category-wrapper'>
                            <Icon icon='IcTradetypeTurboslong' className='category-type' color='brand' />
                        </div>
                        <div className='category-wrapper'>
                            <Icon icon='IcTradetypeTurbosshort' className='category-type' color='brand' />
                        </div>
                    </React.Fragment>
                );
                break;`

Reply:
@kate-deriv  like this?

Reply:
Like this:
```
  case 'turboslong':
  case 'turbosshort':
  IconCategory = (
                    <React.Fragment>
                        <div className='category-wrapper'>
                            <Icon icon='IcTradetypeTurboslong' className='category-type' color='brand' />
                        </div>
                        <div className='category-wrapper'>
                            <Icon icon='IcTradetypeTurbosshort' className='category-type' color='brand' />
                        </div>
                    </React.Fragment>
                );
                break;
```